src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  height: $card-height;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+    background-size: 160px 134px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-top: 16px;
+  }
+
+  // &__content {
+  //   margin: 40px;
+  // }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin-top: 40px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    display: flex;
+    justify-content: flex-start;
+    font-size: 10px;
+    line-height: 14px; /* 140% */
+    margin-top: 4px;
+  }
+
+  &__review {
+    color: $main-accent;
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px; /* 140% */
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__buy-button {
+    // width: 166px;
+    // height: 40px;
+    
+    padding: 12px;
+
+    flex-shrink: 0;
+    border-radius: 5px;
+    background: $blue-accent;
+    justify-content: center;
+    display: flex;
+    color: $white-color;
+    text-align: center;
+    align-items: center;
+    font-size: 14px;
+    font-weight: 700;
+
+    line-height: 16px; /* 114.286% */
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      border-radius: 5px;
+      border: 1px solid $blue-accent;
+      background: #FFF;
+
+      color: $blue-accent;
+      text-align: center;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px; /* 114.286% */
+      text-transform: uppercase;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin: 24px 0 16px;
+
+    &-title {
+      color: $secondary-color;
+      font-size: 12px;
+      line-height: 18px; /* 150% */
+    }
+
+    &-sum {
+      color: $main-accent;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px; /* 112.5% */
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    transition-duration: $transition;
+  }
+
+  &:hover &__title {
+    color: $title-hover;
+    transition-duration: $transition;
+  }
```

Comment:
Hover effects should be smooth for both cases when mouse enters on card and when it leaves